CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# CONTRIBUTING
+Contributions to grpc-json-client are welcome by all!
+## Build Environment Setup
+### Dependencies
+1. Install [CMake](https://cmake.org/) and add it to `path`.
+2. Install [gRPC](https://github.com/grpc/grpc) by following the instructions [here](https://github.com/grpc/grpc/blob/master/BUILDING.md).
+### Getting the Source
+1. Clone the repo:
+```
+> git clone https://github.com/ni/grpc-json-client.git
+> cd grpc-json-client
+```
+2. Initialize submodules:
+```
+> git submodule update --init
+```
+### Building on Windows
+1. Create a build directory:
+```
+> mkdir build
+> cd build
+```
+2. Generate CMake cache:
+```
+> cmake .. "-DCMAKE_PREFIX_PATH=C:\Program Files (x86)\grpc"
+``` 
+3. Build with Debug configuration:
+```
+> cmake --build . --config Debug
+```
+### Tips
+The "Desktop development with C++" workload for Visual Studio includes IDE support for CMake. A `CmakeSettings.json` file is provided that contains the default build configuration.
```

Comment:
Your CMakeSettings.json uses Ninja (great!), does that mean Ninja is also required in the default config or is that an optional dep? I thought on windows you needed the `-A x64` flag ([link](https://github.com/ni/grpc-device/blob/6f720619a7f9cafec5ff753ce9c05caebcf1153e/.github/workflows/build_cmake.yml#L28)) to build with msvc (Ninja doesn't need it)?

Setting up a GH workflow (or some other containerized build) would help hammer this down. `grpc-device` may be an anti-example there. But something like [this simple one](https://github.com/astarche/grpc-daq-cpp-demo/blob/main/.github/workflows/build.yml) is pretty handy.

Reply:
Visual Studio actually set up ninja for me when i created the CMake project.  I'm actually unsure how to get the build to use ninja without the Developer Command Prompt for VS 2022.